metrics/seata-metrics-core/src/main/java/io/seata/metrics/exporter/ExporterType.java

```diff
@@ -26,7 +26,17 @@
     /**
      * Export metrics data to Prometheus
      */
-    PROMETHEUS;
+    PROMETHEUS("prometheus");
```

Comment:
Just a question about this kind modification. Why not ignore case in EnhancedServiceLoader#load, keeping the original enum implementation unchanged?

Reply:
First, name will be used as a judgment point to distinguish whether two extensiondefinitions are the same，more over I want users to use our spi in a more rigorous and standard way